docker/docker-compose.yml

```diff
@@ -25,8 +27,59 @@ services:
       - GPC_CONSUMER_SSP_URL
       - GPC_ENABLE_SDS
   gpcc-mocks:
+    networks:
+      - commonforgpc
     build:
       context: ../
       dockerfile: docker/gpcc-mocks/Dockerfile
     ports:
-      - "8110:8080"
\ No newline at end of file
+      - "8110:8080"
+  gpconnect-db:
+    image: mariadb
+    networks: 
+      - commonforgpc
+    environment:
+      - MYSQL_ROOT_PASSWORD=password
```

Comment:
should we be storing the root password? Is this needed?

Reply:
That's how NHS have set it up, nothing really on our side we can change